test/toTransform.js

```diff
@@ -0,0 +1,17 @@
+/* global suite test assert toTransform InvalidArgument */
+
+suite('toTransform', function () {
+  test('testTranslate', function () {
+    assert.throws(function () {
+      toTransform({translate: ''});
+    }, InvalidArgument);
+
+    assert.equal(toTransform({translate: 'none'}), '');
+
+    assert.throws(function () {
+      toTransform({translate: '20px 30px 27px 90px'});
+    }, InvalidArgument);
```

Comment:
We should group these assert.throws checks together to make this easier to read.